### ISSUE TITLE
A11y_SQL Azure Data Studio_NewTables_Cloumns_Info & Relationship: NVDA doesn't announce the selected item as selected in Type combobox.

### ISSUE THREAD

**Issue by ShivamPandey-V** [microsoft/azuredatastudio#19412]
**GitHub Tags:**
#A11yMAS;#A11yTCS;#Win11;#WCAG1.3.1;#DesktopApp;#A11ySev2;#NVDA;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

## Environmental Details:
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 11

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till column and press enter.
6.Check the type column 3 selected item is announced as Selected or not.
## Note:
Same issue found for all the dropdown

## Actual results:
NVDA doesn't announce the selected item as selected in Type Combo box.

## Expected results:
NVDA should announce the selected item as selected in Type combo box.

## User Impact:
Users will get confuse to differentiate whether the item is selected or not.

## Attachments:
https://user-images.githubusercontent.com/100270292/169005942-5f73657b-af14-432d-a9fe-7103f62fabbc.mp4

**Comment by Vishnuvarthancr7** [comment]
#Closed;
